prime-router/metadata/fhir_mapping/hl7/extension/ExtensionAnnotation.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I do not think you need this config.  I am pretty sure you can just do something like this.  To see all the types of values supported you can look at the Extension class.
```
extension_1:
  generateList: true
  expressionType: nested
  expressionsMap:
    url:
      type: SYSTEM_URL
      value: ???
    valueAnnotation:
      valueOf: datatype/Annotation
      expressionType: resource
      specs: NTE
```

Reply:
yup 6 of one half a dozen of another, I can remove and use the map pattern